samples/Azure.Management.Storage/Generated/LongRunningOperation/StorageArmOperation.cs

```diff
@@ -31,9 +31,9 @@ internal StorageArmOperation(Response response)
             _operation = new OperationOrResponseInternals(response);
         }
 
-        internal StorageArmOperation(ClientDiagnostics clientDiagnostics, HttpPipeline pipeline, Request request, Response response, OperationFinalStateVia finalStateVia)
+        internal StorageArmOperation(ClientDiagnostics clientDiagnostic, HttpPipeline pipeline, Request request, Response response, OperationFinalStateVia finalStateVia)
```

Comment:
Is it intentional?

Reply:
Ah, I already noticed this change, and has changed it. I did not regenerate this project yet. If you take a look at the Azure.ResourceManager.Sample project, this change is not there.
I will refresh all the test project shortly